### ISSUE TITLE
Latest Windows installer: msfconsole can't be started due to a missing dependency (requiring bcrypt 3.1.15 but 3.1.16 is installed)

### ISSUE THREAD

**Issue by franferrax** [rapid7/metasploit-framework#14101]
## Steps to reproduce
1. Download the latest *Windows* `*.msi` installer from https://windows.metasploit.com
    * NOTE: I reproduced it with `metasploit-framework-6.0.5+20200905152636-1rapid7-1-x64.msi` but it is likely to be present in any `metasploit-framework-6.0.5+*-1rapid7-1-x64.msi` installer
1. If possible, use a clean *Windows* virtual machine (I reproduced it in *Windows 10 Enterprise, version 1903*)
1. Install *Metasploit Framework* with the following commands in an **elevated** command line:
    ```bat
    PowerShell "Add-MpPreference -ExclusionPath 'C:\metasploit-framework'"
    MsiExec -i "metasploit-framework-6.0.5+20200905152636-1rapid7-1-x64.msi" /quiet /passive
    ```
1. Launch the *Metasploit Framework* console
    ```bat
    C:\metasploit-framework\bin\msfconsole.bat
    ```

NOTE: no exploit nor victim host indicated sine this issue occurs early when starting the console.

## Expected behavior
The console finishes loading and the `msf6 >` prompt is shown.

## Current behavior
The following error is reported:
```
Could not find bcrypt-3.1.15 in any of the sources
Run `bundle install` to install missing gems.
```

### Metasploit version
```
Framework: 6.0.5-dev-ce17c434ae6ba29f9facec3fc449ae109eda6f00
Console  : 6.0.5-dev-ce17c434ae6ba29f9facec3fc449ae109eda6f00
```

## Additional Information
I checked the installed version of `bcrypt` with the following commands:
```bat
set PATH=C:\metasploit-framework\embedded\bin;%PATH%
gem list | findstr /C:bcrypt
```
It turned out to be `3.1.16`:
```
bcrypt (3.1.16)
bcrypt_pbkdf (1.0.1 x64-mingw32)
```

However, I found that `bcrypt (3.1.15)` is 'pinned' in [Gemfile.lock:145](/rapid7/metasploit-framework/blob/d8447e97083ce395fc4a09a460a627246b8cb112/Gemfile.lock#L145). I tried by manually editing `C:\metasploit-framework\embedded\framework\Gemfile.lock` to specify `bcrypt (3.1.16)` and it worked.

I noticed that `Gemfile.lock` is updated by a *Jenkins Job* (see d8447e97083ce395fc4a09a460a627246b8cb112) but I'm afraid I'm very unfamiliar with the *Ruby* language and its package dependency system, so I don't know how or where to fix this.

This issue might be related with #13933 & #13894.

### Workaround
As I previously mentioned, we can edit `Gemfile.lock` as a temporary workaround, in the command line:
```bat
set gfl=C:\metasploit-framework\embedded\framework\Gemfile.lock
PowerShell "((Get-Content $Env:gfl -Raw) -replace 'bcrypt \(3\.1\.15\)', 'bcrypt (3.1.16)') | Set-Content $Env:gfl"
```

**Comment by GetRektBoy724** [comment]
its better to not install metasploit framework on windows

**Comment by GetRektBoy724** [comment]
you better install it on kali wsl

**Comment by franferrax** [comment]
> its better to not install metasploit framework on windows
> you better install it on kali wsl

I understand, I can't use another OS but I'm not blocked since I can use version `6.0.4` or change `Gemfile.lock` as a temporary workaround. I just wanted to report this because `6.0.4` was working in *Windows*.

**Comment by franferrax** [comment]
Apparently, this got fixed in 1b77d01f2317d4d147c2fd5251e442d44c3e1192, from #14096, which was merged 2 days ago. I'll verify and close this issue when a new *Windows* installer is generated that includes those changes. The current one (`metasploit-framework-6.0.5+20200910152637-1rapid7-1-x64.msi`) was built from 9a5f393e0b1a34b24a2f9eb715c89d4d99816d0c so it doesn't include them yet.

**Comment by franferrax** [comment]
Since the `msfconsole` is working properly with `metasploit-framework-6.0.7+20200913152640-1rapid7-1-x64.msi`, I'm closing this issue.
